wallet/wallet.go

```diff
@@ -627,6 +627,21 @@ func (w *Wallet) loadActiveAddrs(ctx context.Context, dbtx walletdb.ReadTx, nb N
 	return bip0044AddrCount + importedAddrCount, nil
 }
 
+// CoinTypePrivKey returns the BIP0044 coin type private key.
+func (w *Wallet) CoinTypePrivKey() (*hdkeychain.ExtendedKey, error) {
+	const op errors.Op = "wallet.CoinTypeKey"
```

Comment:
Wrong name

Reply:
@jrick fixed the op to CoinTypePrivKey if this is what you were referring too.